style.css

```diff
@@ -0,0 +1,85 @@
+@import url("https://fonts.googleapis.com/css?family=Poppins:200,300,400,500,600,700,800,900&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+  background-color: #e5e5e5;
+}
+
+a {
+  text-decoration: none;
+  cursor: pointer;
+}
+
+li {
+  list-style-type: none;
+}
+
+/* intro style */
+.container-fluid {
+  position: relative;
+  top: 0;
+  bottom: 0;
+  width: 100%;
+  background-color: #fff;
+  border-bottom-left-radius: 60px;
+  box-shadow: 1px 0 1px #fff;
+}
+
+/* logo style */
+.display-flex1 {
+  display: flex;
```

Comment:
- [ ] For DRY css you should combine duplicate properties. 
You can create class `.d-flex { display: flex}` and use this class in html file.

- For more information about DRY principles you can check this [article](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- Also you can refresh your knowledge about "html and css best practices" in [curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

Reply:
Done